### ISSUE TITLE
How to make this work on a phpbb style?

### ISSUE THREAD

**Issue by master412160** [davatron5000/Lettering.js#61]
I've tried everything that I can possible think of. Including just copying the js and html code examples given in the exmaple folder but none of it works on a phpbb style.

**Comment by davatron5000** [comment]
This may be more of a StackOverflow question since it's about a particular implementation. We're limiting Github issues to actual bugs. 




Have you tried doing this outside of PHPbb in something like a Codepen? Without a reduced test case I'll be forced to close the issue.

**Comment by master412160** [comment]
Sire http://codepen.io/anon/pen/ZYNYBO  but the codepen doesn't seem to load...

**Comment by TrentWalton** [comment]
Forked the codepen and confirmed test case works: http://codepen.io/TrentWalton/pen/LEoRpz

Used your hosted version of Lettering (see settings gear in JS panel)

That’s probably as far as we can take it. Good luck!

**Comment by master412160** [comment]
Could you tell me why then my code doesn't work compared to yours? 

I've added the html; http://phpbb.miscy.net/index.php called rainbows

Got the js file linked properly in the head tag.

And added

**Comment by Binero** [comment]
I can confirm this isn't working. It seems the `.lettering` function isn't correctly being defined on such pages. Other jQuery functions work. Loading the actual script works with HTTP 300, so that should all be good. 

It is hard to deduce what is the exact difference between these pages and a test page.

**Comment by TrentWalton** [comment]
And I actually don’t see this anywhere on the page at http://phpbb.miscy.net/index.php

```
<script>
//Lettering.js
$(function() {
    $("#demo1 h1").lettering();
    });

</script>
```

**Comment by davatron5000** [comment]
1. You including two versions of jQuery on the page. One in the `<head>` and one before the closing `</body>` tag. You should only ever have 1 copy of jQuery.
1. As @TrentWalton pointed out, you're never actually calling the `.lettering()` function it seems. You need to write that JavaScript to make it work correctly.

You have a few problems with your code. My original recommendation still stands: Take this question to [StackOverflow](http://stackoverflow.com/), they are a community of people dedicated to helping with code problems. 

It would also be good to learn how jQuery works. I really like the talk "Fixing These jQuery":

Video: https://vimeo.com/25698808
Slides: http://fixingthesejquery.com/#slide1

Good luck! I'm closing this issue because it is not a bug with our code.

**Comment by master412160** [comment]
I had a guy help me out who knows jquery, the guy that replies here is that person.

He made me test out were the issue lies. And he's conclusion is that your plugin doesn't work.

he tested this by adding: 

'''<script>
$(document).ready(function() {
    $(".demo h1").fadeOut(10000);
});
</script>

which works so no jquery issue or conflict present.

But when I add:

<script>
//Lettering.js
$(function() {
    $("#demo1 h1").lettering();
    });

</script>

It doesn't work. So that is why you didn't see the code present. I had replaced it with thefadeout one and that works so no jquery issue just with your plugin.

Also I had moved the jquery defined file to the body instead of the head for further testing. I'll move it to the head. But I don't see jquery defined twice. No errors.
